site/src/components/PopoverContainer/PopoverContainer.tsx

```diff
@@ -0,0 +1,149 @@
+/**
+ * @file Abstracts over MUI's Popover component to simplify using it (and hide)
+ * some of the wonkier parts of the API.
```

Comment:
Might be useful to hint at some of the wonkier API parts of MUI, or maybe link to their docs.

Reply:
Yeah, good point. I guess if I had to summarize up my concern, it feels to me like their current API (as in, the entire conceit of it) defeats the point of React?

Like, the point of JSX is that you don't have to touch live DOM nodes during actual render logic, and only have to worry about them when you use effects and event handlers as escape hatches. But MUI has you take a whole real DOM node (which is physically incapable of existing on the first render), and store it in React state, to be accessed in renders

Reply:
> it feels to me like their current API (as in, the entire conceit of it) defeats the point of React?
lolol

Your abstraction makes sense to me. Maybe you could make a contribution!